git_machete/cmd.py

```diff
@@ -2606,34 +1933,27 @@ def is_merged_to(machete_client: MacheteClient, cli_ctxt: CommandLineContext, b:
 
 
 def get_revision_repr(cli_ctxt: CommandLineContext, revision: str) -> str:
```

Comment:
Ditto

Reply:
Btw, I've got an impression that in 90% of methods that accept `CommandLineContext`, it's only ever used to access `opt_debug`... actually I'd consider making `opt_debug` a global variable again to simplify things 😅 to be done on a separate PR